ui/css/utilities/colors.scss

```diff
@@ -9,6 +9,10 @@ Before adding a color here make sure that there isn't a design token available.
   --mainnet: #29b6af;
   --inherit: inherit;
   --transparent: transparent;
+  --color-network-goerli-default: #1098fc;
+  --color-network-sepolia-default: #cfb5f0;
+  --color-network-goerli-inverse: #fcfcfc;
+  --color-network-sepolia-inverse: #fcfcfc;
```

Comment:
These third party network colors have been removed from v4 of the design tokens. There was some previous thoughts that we could manage test network colors within the MetaMask brand as test networks generally don't have any branding elements attached, but ultimately they are third party entities and should be treated as such. We should treat test networks the same as other networks and use images (even if we are the ones creating those images). Eventually these should be replaced. 

From design tokens v4 [migration docs](https://github.com/MetaMask/design-tokens/blob/main/MIGRATION.md#removed)

<img width="1046" alt="Screenshot 2024-06-09 at 9 11 11 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d0696784-b721-47b0-a9f7-7e5c35ce989f">

 I have created a ticket to remove these colors here https://github.com/MetaMask/metamask-extension/issues/24969
 
 Move this to the design token v4 branch so we can ensure that they are still working in that PR
 
 Network colors still work after upgrade

<img width="1511" alt="Screenshot 2024-06-09 at 9 13 52 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/02ef2b03-c918-4286-b73f-e9469712686f">